examples/fitness/lib/settings.dart

```diff
@@ -30,6 +45,64 @@ class SettingsFragment extends Component {
     );
   }
 
+  String get goalWeightText {
+    if (userData.goalWeight == null || userData.goalWeight == 0.0)
+      return "None";
+    else
+      return "${userData.goalWeight}";
+  }
+
+  static final GlobalKey weightGoalKey = new GlobalKey();
+
+  double _goalWeight;
+
+  void _handleGoalWeightChanged(String goalWeight) {
+    // TODO(jackson): Looking for null characters to detect enter key is a hack
```

Comment:
Might reference https://github.com/domokit/sky_engine/issues/574 in the comment.